shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -363,8 +365,12 @@ public void sendTextInputAppPrivateCommand(String action, Bundle data) {
   }
 
   private void showTextInput(View view) {
-    view.requestFocus();
-    mImm.showSoftInput(view, 0);
+    if (configuration.inputType.type != TextInputChannel.TextInputType.NONE) {
```

Comment:
I would recommend not creating an input connection at all if you don't need the keyboard. See: https://developer.android.com/reference/android/view/View#onCreateInputConnection(android.view.inputmethod.EditorInfo)